pkg/services/pluginsintegration/pluginac/accesscontrol.go

```diff
@@ -1,4 +1,4 @@
-package plugins
+package pluginac
```

Comment:
nit for readability, I would call this package `accesscontrol`, `pluginac` is difficult to know what it means

Reply:
Yeah maybe I'm just conditioned to seeing `ac` across some of the existing packages. 

The only thing I think with using `accesscontrol` is that at a glance it could be mixed up with the core access control package, and then `pluginaccesscontrol` may be too long 😞 I'll see what other people think too

Reply:
I see, in this case I would error in being too verbose (so `pluginaccesscontrol` sounds good to me)